pom.xml

```diff
@@ -126,6 +127,11 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
+			<dependency>
```

Comment:
I think this is not needed and anyway version in it was wrong.

Reply:
This is required in dependency management else the version has to specified in the SingleStepBatchJob pom.    This dependency is not a part of the task starter.

Reply:
Ah I didn't realise it's part of a spring-cloud-task.